packages/cdk/lib/env/context-app-parameters.ts

```diff
@@ -79,6 +79,12 @@ export class ContextAppParameters {
    * The types of EC2 instances that may be launched in the compute environment.
    */
   public readonly instanceTypes?: InstanceType[];
+  /**
+   * If true, put EC2 instances into public subnets instead of private subnets. This should be used in conjunction
+   * with the ENV_PUBLIC_SUBNETS option
+   * This will result in significantly lower ongoing costs when no job is running.
```

Comment:
Same as above
due to NAT gateways ...

Probably also worth documenting the potential security risk if users also modify security groups

Reply:
I would like to avoid repeating myself where possible. Can I put this discussion in the documentation?